python/fastsim/__init__.py

```diff
@@ -29,11 +29,10 @@ def package_root() -> Path:
 logger = logging.getLogger(__name__)
 
 from pkg_resources import get_distribution
-
 __version__ = get_distribution("fastsim").version
 
-__doc__ += "\nhttps://pypi.org/project/fastsim/"
-__doc__ += "\nhttps://www.nrel.gov/transportation/fastsim.html"
+# __doc__ += "\nhttps://pypi.org/project/fastsim/"
```

Comment:
@robinsteuteville , these were actually appending these lines to the module level doc string, which is something we'd definitely like to retain.  Please uncomment.